From 00579a38cd37d0723018b2b0adbd5c4b4571abac Mon Sep 17 00:00:00 2001
From: Karthikeyan K <karthikk@habana.ai>
Date: Tue, 15 Mar 2022 13:00:47 +0200
Subject: [PATCH] [SW-0] rom: align regs name with specs

MEM_CPY AXUSER register has been renamed for gaudi3 in latest
global specs update. This patch aligns the regmap with the same.

Change-Id: I49aa36c96d2883bb1d5ed7d0997acc236202481f
---

diff --git a/arch/gaudi3/arc/regmap.h b/arch/gaudi3/arc/regmap.h
index 3cb8d50..77387c2 100644
--- a/arch/gaudi3/arc/regmap.h
+++ b/arch/gaudi3/arc/regmap.h
@@ -36,7 +36,7 @@
 #define mmMEM_CPY_DEST_ADDR_H		mmD0_PSOC_GLOBAL_CONF_MEM_CPY_DEST_ADDR_H
 #define mmMEM_CPY_DEST_ADDR_L		mmD0_PSOC_GLOBAL_CONF_MEM_CPY_DEST_ADDR_L
 #define mmMEM_CPY_CTRL2			mmD0_PSOC_GLOBAL_CONF_MEM_CPY_CTRL2
-#define mmMEM_CPY_AXUSER		mmD0_PSOC_GLOBAL_CONF_MEM_CPY_AXUSER
+#define mmMEM_CPY_AXUSER		mmD0_PSOC_GLOBAL_CONF2_MEM_CPY_HBW_ATTR_CTRL_HB_MMU_BYPASS_OVRD
 #define mmSPI_DIRECT_WR_RD_CTRL		mmD0_PSOC_GLOBAL_CONF_SPI_DIRECT_WR_RD_CTRL
 
 #define mmGPIO0_DATA			mmD0_PSOC_GPIO0_DATA
